docs/10-stories.md

```diff
@@ -0,0 +1,168 @@
+# Stories
+
+Source uses [storybook](https://storybook.js.org/) for its API documentation as well as for visual regression testing.
+
+The Source storybook can be run locally using `yarn storybook`
+
+## Story Configuration
+
+1. Story files should either be names `stories.tsx` or `MyComponent.stories.tsx`
```

Comment:
Should we just have `MyComponent.stories.tsx` everywhere? Does that also rely on having `MyComponent.tsx` everywhere and `index.ts` files that only contain exports (even in packages with a single component)?

Reply:
i think is the approach i'd favour. it's also where we plan to be in v4 i think? i've taken this approach so far with the branding and accordion ones and i dont think it was too disruptive

Reply:
I'll open a PR to make this consistent across the components that have already been migrated.

Reply:
How do we feel about components with lots and lots of stories, like `<Radio>`? (see #902) Currently, stories are split into separate files and grouped vaguely by the prop the stories are demonstrating. Do we want to collapse all these stories into a single file?

Reply:
I've started #905 to modify the components that we've already migrated API docs for to conform with these recommendations.

On multiple vs one file, I started moving all of the radio stories into one file (well, one each for `Radio` and `RadioGroup`) and then decided against it but I'm leaning back towards it now. I've also added some card around reducing duplication in stories via [decorators](https://storybook.js.org/docs/react/writing-stories/decorators#story-decorators) and [templates](https://storybook.js.org/docs/react/writing-stories/decorators#story-decorators) which would make the single files less unwieldy.

Reply:
https://github.com/guardian/source/pull/902/commits/af04304ebe1f8a41b00247af892afb6483fd8fa5 in #902 consolidates the `Radio` and `RadioGroup` stories into one file per component

Reply:
👍 from me for one `stories` file per component